src/components/Navigation/Navigation.scss

```
@@ -62,7 +62,7 @@ $nav-max-width: 360px;
 
 .LogoContainer {
   display: none;
-  @include breakpoint-before(nav-min-window-corrected()) {
+  @include breakpoint-before(nav-min-window-corrected() - 1) {
```

Does scss actually compile this? I think it may need to be interpolated i.e. 
```suggestion
  @include breakpoint-before(#{nav-min-window-corrected() - 1}) {
```

I could be wrong though.

Another thing we could do is:

```suggestion
  @include breakpoint-before($breakpoint: nav-min-window-corrected(), $inclusive: false) {
```

Which is a bit easier to understand what's happening IMO

---

Applied in https://github.com/Shopify/polaris-react/pull/5213/commits/4cb9b7b65f8c270d2289cd112a03890feb1b413e